docs/apache-airflow/logging-monitoring/check-health.rst

```diff
@@ -105,7 +105,7 @@ HTTP monitoring for Celery Cluster
 
 You can use Flower to monitor the health of the Celery cluster. It also provides an HTTP API that you can use to build a health check for your environment.
 
-For details about installation, see: :ref:`executor:CeleryExecutor`. For details about usage, see: `The Flower project documentation <https://flower.readthedocs.io/en/stable/>`__.
+For details about installation, see: :ref:`executor:CeleryExecutor`. For details about usage, see: `The Flower project documentation <https://flower.readthedocs.io/en/latest/>`__.
```

Comment:
```suggestion
For details about installation, see: :ref:`executor:CeleryExecutor`. For details about usage, see: `The Flower project documentation <https://flower.readthedocs.io/>`__.
```

According to ReadTheDocs, this should redirect to the correct location, no matter whether Flower chooses to expose stable or latest version or documentation, or even both (they are slightly different).